app/src/main/res/layout/activity_choose_party_room.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/floral_white"
+    tools:context=".activity.ChoosePartyRoomActivity">
+
+    <EditText
+        android:id="@+id/roomNameInput"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="440dp"
+        android:ems="10"
+        android:hint="Party Name"
```

Comment:
Hardcoded string here. It's fine overall, but please remember to fix it in your next PR

Reply:
Noted, I will fix it in the next PR